kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/ContentCardList.vue

```diff
@@ -45,7 +45,7 @@
           :title="content.title"
           :thumbnail="content.thumbnail"
           :description="content.description"
-          :kind="content.kind"
+          :kind="content.is_leaf ? content.learning_activities : 'folder'"
```

Comment:
Let's change the prop name here to indicate that it is no longer accepting a content kind?